code/game/objects/structures/watercloset.dm

```diff
@@ -183,6 +183,20 @@
 	return ..()
 
 /obj/structure/urinal/attackby(obj/item/I as obj, mob/user as mob)
+	if(iswrench(I))
+		to_chat(user, "<span class='notice'>You [anchored ? "un":""]bolt \the [src]'s grounding lines.</span>")
+		anchored = !anchored
+	if(anchored == 0)
```

Comment:
it's not a big deal but I think this is more elegantly expressed as !anchored

Reply:
done